<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QMainWindow" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>932</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>1081</width>
      <height>531</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/field.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-20</y>
      <width>1131</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/JUST_FOR_FILL.png</pixmap>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Sun">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(184, 0, 184);</string>
    </property>
    <property name="text">
     <string>SUN</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_brain">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>110</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(184, 0, 184);</string>
    </property>
    <property name="text">
     <string>BRAIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="remaining_time_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(184, 0, 184)</string>
    </property>
    <property name="text">
     <string>3:30</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="field_table">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>941</width>
      <height>451</height>
     </rect>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>6</number>
    </property>
    <property name="columnCount">
     <number>22</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>42</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>13</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>620</y>
      <width>1131</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/JUST_FOR_FILL.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit_Button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>810</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 0, 0);
color:rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Quit game</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>670</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/astronaut op.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="astro_Z">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>770</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(170, 0, 127)</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>660</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/Bucket head op.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="bucket_Z">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>770</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(170, 0, 127)</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>660</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/leaf hair op.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="leaf_Z">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>770</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(170, 0, 127)</string>
    </property>
    <property name="text">
     <string>150</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>660</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/purple hair op.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>770</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(170, 0, 127)</string>
    </property>
    <property name="text">
     <string>800</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>660</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/regular op.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>770</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(170, 0, 127)</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>660</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/tall op.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="twopeashoot_P">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>770</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(170, 0, 127)</string>
    </property>
    <property name="text">
     <string>150</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/plum mine op.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="astro_Z_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(120, 255, 165);
color:rgb(255, 85, 0)</string>
    </property>
    <property name="text">
     <string>175</string>
    </property>
   </widget>
   <widget class="QPushButton" name="peashoot_P">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>770</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(120, 255, 165);
color:rgb(255, 85, 0)</string>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>660</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/two_peashooter op.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>660</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/peashooter op.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>660</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/boomrang op.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="twopeashoot_P_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>770</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(120, 255, 165);
color:rgb(255, 85, 0)</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>660</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/jalapino op.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="wallnut_P">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>770</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(120, 255, 165);
color:rgb(255, 85, 0)</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Jalo_P">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>770</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(120, 255, 165);
color:rgb(255, 85, 0)</string>
    </property>
    <property name="text">
     <string>150</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boom_P">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>770</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(120, 255, 165);
color:rgb(255, 85, 0)</string>
    </property>
    <property name="text">
     <string>125</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>660</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/opimized pictures/walnut op.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>830</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>collected_suns</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>820</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>collected_brains</string>
    </property>
   </widget>
   <zorder>field_table</zorder>
   <zorder>label_6</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>progressBar_Sun</zorder>
   <zorder>label_3</zorder>
   <zorder>progressBar_brain</zorder>
   <zorder>label_4</zorder>
   <zorder>remaining_time_label</zorder>
   <zorder>Quit_Button</zorder>
   <zorder>label_5</zorder>
   <zorder>astro_Z</zorder>
   <zorder>label_7</zorder>
   <zorder>bucket_Z</zorder>
   <zorder>label_8</zorder>
   <zorder>leaf_Z</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>label_10</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>label_11</zorder>
   <zorder>twopeashoot_P</zorder>
   <zorder>label_12</zorder>
   <zorder>astro_Z_2</zorder>
   <zorder>peashoot_P</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>label_15</zorder>
   <zorder>twopeashoot_P_2</zorder>
   <zorder>label_16</zorder>
   <zorder>wallnut_P</zorder>
   <zorder>Jalo_P</zorder>
   <zorder>boom_P</zorder>
   <zorder>label_17</zorder>
   <zorder>label_18</zorder>
   <zorder>label_19</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
